tests/test_alignments.py

```diff
@@ -13,8 +14,97 @@
     else os.path.join(root_path, "3rd_party", "marian-dev", "build")
 )
 
+# "|||" in the text can cause issues if joint fast_align style input is used
+en_sample = """The little girl, seeing she had lost one of her pretty shoes, grew angry, and said to the Witch, “Give me back my shoe!” ||| one
+“I will not,” retorted the Witch, “for it is now my shoe, and not yours.”
+“You are a wicked creature!” cried Dorothy. “You have no right to take my shoe from me.”
+“I shall keep it, just the same,” said the Witch, laughing at her, “and someday I shall get the other one from you, too.”
+This made Dorothy so very angry that she picked up the bucket of water that stood near and dashed it over the Witch, wetting her from head to foot.
+Instantly the wicked woman gave a loud cry of fear, and then, as Dorothy looked at her in wonder, the Witch began to shrink and fall away.
+“See what you have done!” she screamed. “In a minute I shall melt away.”
+“I’m very sorry, indeed,” said Dorothy, who was truly frightened to see the Witch actually melting away like brown sugar before her very eyes.
+"""
 
-def test_alignments_and_shortlist():
+ru_sample = """Маленькая девочка, увидев, что потеряла одну из своих красивых туфелек, рассердилась и сказала Ведьме: «Верни мне мою туфельку!» ||| один
+«Я не буду, — парировала Ведьма, — потому что теперь это моя туфля, а не твоя».
+«Ты злое существо!» - воскликнула Дороти. «Ты не имеешь права забирать у меня туфлю».
+«Я все равно сохраню его, — сказала Ведьма, смеясь над ней, — и когда-нибудь я получу от тебя и другой».
+Это так разозлило Дороти, что она взяла стоявшее рядом ведро с водой и облила им Ведьму, обмочив ее с головы до ног.
+Мгновенно злая женщина громко вскрикнула от страха, а затем, когда Дороти с удивлением посмотрела на нее, Ведьма начала сжиматься и падать.
+«Посмотри, что ты наделал!» она закричала. «Через минуту я растаю».
+«Мне действительно очень жаль», — сказала Дороти, которая была по-настоящему напугана, увидев, что Ведьма тает, как коричневый сахар, у нее на глазах.
+"""
+
+
+def verify_aln(data_dir, dataset, src_corpus, trg_corpus):
```

Comment:
Suggestion: Please spell out alignment here, it's a bit more to type, but it makes the code easier to read if we avoid acronyms that others may have trouble interpreting.